manifests/repo.pp

```diff
@@ -54,8 +54,9 @@
       }
     }
     'RedHat': {
-      $libcontainers_url = "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/CentOS_${fact('os.release.major')}/"
-      $crio_url          = "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/${crio_version}/CentOS_${fact('os.release.major')}/"
+      $release_name = if fact('os.release.major') > 7 { "CentOS_${fact('os.release.major')}_Stream" } else { "CentOS_${fact('os.release.major')}" }
```

Comment:
You can not compare String with Integer.
Please use versioncmp Function instead.

Reply:
Error message from Puppet: `Error: Evaluation Error: Comparison of: String < Integer, is not possible. Caused by 'A String is not comparable to a non String'.`

Reply:
Sorry about that. I thought that the static validation would have caught it. The `docker build .` doesn't catch that either. The `bundle exec rake release_checks` fails, but it may be because of the documentation warnings. I tested the change with Puppet this time.